app/bundles/PageBundle/Model/PageModel.php

```diff
@@ -804,28 +804,56 @@ public function getHitsLineChartData($unit, \DateTime $dateFrom, \DateTime $date
      * Get data for pie chart showing new vs returning leads.
      * Returning leads are even leads who visits 2 different page once.
      *
-     * @param \DateTime $dateFrom
-     * @param \DateTime $dateTo
-     * @param array     $filters
-     * @param bool      $canViewOthers
+     * @return array<string, array<int|string>>
      */
-    public function getNewVsReturningPieChartData($dateFrom, $dateTo, $filters = [], $canViewOthers = true): array
```

Comment:
Renaming a public method is a BC break. Please keep the method there, deprecate it and call the new function from the inside.

Reply:
Ah, I forgot push commits from last week. It's done